lib/Cache/RuntimeCacheTrait.php

```diff
@@ -18,26 +18,42 @@
 use Exception;
 use Pimcore\Cache;
 
+/**
+ * @deprecated
+ */
 trait RuntimeCacheTrait
 {
     /**
      * @var bool
+     * @deprecated
      */
     private static bool $cacheEnabled = false;
 
     /**
      * @param bool $cacheEnabled
+     * @deprecated
      */
     public static function setCacheEnabled(bool $cacheEnabled): void
     {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '10.5.0',
+            sprintf('%s is deprecated and will be removed in Pimcore 11. Use \Pimcore\Cache\Runtime::set() instead.', __METHOD__)
```

Comment:
Please remove "Use \Pimcore\Cache\Runtime::set() instead."

Reply:
Thanks! This is resolved now!